spec/features/proposal/access_permissions/no_permissions_for_draft_mmt_spec.rb

```diff
@@ -158,3 +158,38 @@
     end
   end
 end
+
+describe 'No Permissions for Non-NASA Workflow in MMT', js: true do
+  before do
+    login
+  end
+
+  context 'when on the manage collections page' do
+    before do
+      visit manage_collections_path
+    end
+
+    it 'the user cannot see the Manage Proposals tab' do
+      expect(page).to have_link('Manage Collections')
+      expect(page).to have_link('Manage Variables')
+      expect(page).to have_link('Manage Services')
+      expect(page).to have_link('Manage CMR')
+      expect(page).to have_no_link('Manage Proposals')
+    end
+  end
+
+  context 'when trying to access the manage proposals page directly' do
+    before do
+      visit manage_proposals_path
+    end
+
+    it 'the user cannot access the Manage Proposals tab' do
+      expect(page).to have_link('Manage Collections')
```

Comment:
I would maybe add a check for the highlighting under Manage Collections to verify that's where we got redirected to